_includes/footer.html

```diff
@@ -1,3 +1,4 @@
+<link rel="stylesheet" href="/css/footer.css">
```

Comment:
This shouldn't be a separate stylesheet. Instead put it in the `main.sass` file that already exists and does styling for the footer and everything else that appears on every page.